gmso/tests/test_site.py

```diff
@@ -93,3 +95,16 @@ def test_add_connection_non_member(self):
             site3.add_connection(bond1)
             site2.add_connection(bond3)
             site1.add_connection(bond2)
+
```

Comment:
You might already have this, but it would be nice to test for an invalid `position`.
```suggestion
def test_invalid_position_size(self):
    with pytest.raises(GMSOError) as e:
        site1 = Site(name=site, position=[0.0,0.0]
```